cypress/integration/vitals.js

```diff
@@ -0,0 +1,59 @@
+function nthChild(CyElement, Iterator) {
+    if (Iterator < 1 ){
+        return undefined
+    }
+    if (Iterator !== 1) {
+        return nthChild(CyElement.next(), Iterator - 1)
+    }
+    return CyElement.first()
+}
+
+function getNthPointOf(Item, Number) {
+    return nthChild(cy.contains(Item).next().children(),Number)
+}
+
+function checkNumberOfPoints(Item, ExpectedInit, ExpectedFill, Scale) {
+    var CurrentItem=cy.contains(Item).next().children()
+    var i = 1
+    while ('UP DOG'){
+        if (i<=ExpectedFill) {
+            CurrentItem.first().should('have.class', (i<=ExpectedInit) ? 'init' : 'fill')
+        }
+        else {
+            CurrentItem.first().should('not.have.class','fill').or('not.have.class','init')      
+        }
+        if (i !== Scale){
+            break
+        }
+        CurrentItem=CurrentItem.next()
+        i++
+    }
+}
+
+describe('Vitals test',()=>{
+    it('Page Exists', () => {
+        cy.visit('/')
```

Comment:
hmmm, co ziskame tymto testom?

Reply:
teoreticky? nic
prakticky? sa mi asi 2x stalo ze som nespecifikoval stranku a cudoval sa preco neprechadzaju testy, a po druhy krat sa mi stalo ze som nieco zkonil v testoch
vo vysledku vypadaju tieto 2 pripady stejne
aspon teda ked to clovek nestuduje tym rozhranim ale pusti si to v terminalu